[2022-11-07 00:30:10] HTTPConnectionPool(host='2005.10.09', port=80): Max retries exceeded with url: /api/v1/Request/tv?ApiKey=XXXXXXXXXXXXXX (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001AEBA748D60>: Failed to establish a new connection: [Errno 11001] getaddrinfo failed'))

